tool/tctl/common/user_command.go

```diff
@@ -272,6 +272,11 @@ func (u *UserCommand) Add(ctx context.Context, client auth.ClientI) error {
 	user.SetRoles(u.allowedRoles)
 
 	if err := client.CreateUser(ctx, user); err != nil {
+		if trace.IsAlreadyExists(err) {
+			usersUpdateCmd := fmt.Sprintf("> tctl users update %v --set-roles %v # replace roles",
+				u.login, strings.Join(u.allowedRoles, ","))
+			fmt.Printf("NOTE: To update an existing local user:\n%v\n\n", usersUpdateCmd)
```

Comment:
Should we combine this into a single `Printf` rather than using `Sprintf` to format a string that we insert into another `Printf`?